test/tests/functional/pbs_checkpoint.py

```diff
@@ -49,11 +49,9 @@ class TestCheckpoint(TestFunctional):
 
     def test_checkpoint_abort_with_preempt(self):
         """
-        This test verifies that checkpoint_abort works as expected by
-        1. setting preempt_order in sched_config to 'C'.
-        2. submitting a job to the default queue.
-        3. submitting a job to the express queue.
-        4. confirming that the first job was checkpointed.
+        This test verifies that checkpoint_abort works as expected when
+        preempt_order in sched_config is set to 'C' and a job in express
+        queue preempts a job in the default queue.
```

Comment:
I'd rephrase this.  How about 'when a job is preempted via checkpoint'.  That is what you are really testing.  There are other ways to cause it to happen than what you are doing which would test the same thing.  The real thing you are testing is preemption via checkpointing.